packages/styles/src/mixins/_color.scss

```diff
@@ -24,12 +24,12 @@
 // Selector for styles to be applied on dark backgrounds
 @mixin on-dark-background($contains: false) {
   @if $contains == true {
-    &:is(#{color.$dark-backgrounds}),
-    :is(#{color.$dark-backgrounds}) & {
+    &:where(#{color.$dark-backgrounds}),
+    :where(#{color.$dark-backgrounds}) & {
```

Comment:
Big change, this was necessary to be able to keep style for button with specific color background (e.g. `.btn-petrol-bright`). This mixin was changed recently to support `post-aler[type"..."]` and this change the specificity as it adds one element to the list of classes, and then override the style for the specific color background.